web/html/src/manager/images/image-profile-edit.js

```diff
@@ -351,15 +341,12 @@ class CreateImageProfile extends React.Component {
     const select = <FormGroup>
       <Label className="col-md-3" name={t("Custom Info Values")}/>
       <div className="col-md-6">
-        <select value="" onChange={(e) => this.addCustomData(e.target.value)}
-          className="form-control">
-          <option key="0" value="" disabled="disabled">{t("Select a custom info key")}</option>
-          {
+        <ReactSelect value="" onChange={({value}) => this.addCustomData(value)}
```

Comment:
When used directly, this won't have the added styling. Would this be a problem?

Reply:
what do you mean? the added `zIndex` style on the menu? would be impacting only in some cases depending on the components that are right below the Select. I had to add the zIndex to higher than 2 otherwise the menu could be hidden by a date picker.

Reply:
Yes, that one, and maybe more stuff in the future. IIUC, you use it directly to avoid the model handling, am I right?

Reply:
yes, and because that one replaces a standard `<select>`, not our component